src/components/calcite-stepper-item/calcite-stepper-item.tsx

```diff
@@ -161,7 +161,7 @@ export class CalciteStepperItem {
     }
   }
 
-  @Listen("calciteStepperItemChange", { target: "parent" })
+  @Listen("calciteStepperItemChange", { target: "body" })
```

Comment:
Imagine we'll need to check that the received event is coming from the parent element here.

Reply:
Yep. Who can refactor this @macandcheese?

Reply:
✋

Reply:
Do you want me to do that separately from this PR?